docs/javascript-api.md

```diff
@@ -280,8 +280,9 @@ Memory.protect(ptr("0x1234"), 4096, 'rw-');
     `Memory.readS64(address)`, `Memory.readU64(address)`,
     `Memory.readShort(address)`, `Memory.readUShort(address)`,
     `Memory.readInt(address)`, `Memory.readUInt(address)`,
-    `Memory.readLong(address)`, `Memory.readULong(address)`:
-    read a signed or unsigned 8/16/32/64-bit/etc. value from `address` and
+    `Memory.readLong(address)`, `Memory.readULong(address)`,
+    `Memory.readFloat(address)`, `Memory.readDouble(address)`:
+    read a signed or unsigned 8/16/32/64-bit/etc. or float/double value from `address` and
```

Comment:
Could you wrap the lines at 80 cols here? It doesn't matter for the final output; it's just nice to keep the markdown readable in raw form.